src/main/java/ch/wisv/areafiftylan/products/controller/TicketRestController.java

```diff
@@ -79,9 +79,9 @@ public TicketRestController(TicketService ticketService, OrderService orderServi
     @PreAuthorize("isAuthenticated()")
     @RequestMapping(value = "/tokens", method = RequestMethod.GET)
     public ResponseEntity<?> getTicketTokensOpenForTransfer(Authentication auth) {
-        UserDetails currentUser = (UserDetails) auth.getPrincipal();
+        User currentUser = (User) auth.getPrincipal();
         Collection<TicketTransferToken> tokens =
-                ticketService.getValidTicketTransferTokensByUser(currentUser.getUsername());
+                ticketService.getValidTicketTransferTokensByUser(currentUser.getEmail());
```

Comment:
Method name is wrong here (was, still is). Should either accept a `User`, or be changed to `getValidTicketTransferTokensByUserEmail()`